[
    {
        "url": "https://api.github.com/repos/MkFoster/lex-dynamodb/issues/comments/520282523",
        "html_url": "https://github.com/MkFoster/lex-dynamodb/issues/1#issuecomment-520282523",
        "issue_url": "https://api.github.com/repos/MkFoster/lex-dynamodb/issues/1",
        "id": 520282523,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyMDI4MjUyMw==",
        "user": {
            "login": "MkFoster",
            "id": 6801366,
            "node_id": "MDQ6VXNlcjY4MDEzNjY=",
            "avatar_url": "https://avatars2.githubusercontent.com/u/6801366?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/MkFoster",
            "html_url": "https://github.com/MkFoster",
            "followers_url": "https://api.github.com/users/MkFoster/followers",
            "following_url": "https://api.github.com/users/MkFoster/following{/other_user}",
            "gists_url": "https://api.github.com/users/MkFoster/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/MkFoster/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/MkFoster/subscriptions",
            "organizations_url": "https://api.github.com/users/MkFoster/orgs",
            "repos_url": "https://api.github.com/users/MkFoster/repos",
            "events_url": "https://api.github.com/users/MkFoster/events{/privacy}",
            "received_events_url": "https://api.github.com/users/MkFoster/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-08-12T02:10:05Z",
        "updated_at": "2019-08-12T02:10:05Z",
        "author_association": "OWNER",
        "body": "Yah, documentation is a bit weak.  Sorry.  This was for a hackathon and is meant to be deployed from the serverless app repository which is an option when you choose to create a new lambda function.\r\n\r\nThat error is probably indicating the Lambda function broke in some way.\r\n\r\nThe lines you have posted above: \"const tableName = process.env.LexLog;\" and \"const primaryKey = process.env.LexLogId;\" won't work because you are referencing environmental variables named \"LexLog\" and \"LexLogId\" you probably didn't set in the Lambda console for your function.  Either change the lines back the way they were and set the correct environmental variables for \"PRIMARY_KEY\" and \"TABLE_NAME\" in your Lambda function's console config or comment out the \"process.env\" lines entirely and hard code the values like this:\r\n`const tableName = 'LexLog';\r\nconst primaryKey = 'LexLogId';`\r\nAssuming \"LexLog\" is the name of your DynamoDB table and \"LexLogId\" is the name of your DynamoDB table primary key.\r\n\r\nIf you are still getting your error, try adding...\r\n`console.log(\"got here!\")`\r\n...to your code to help debug and see how far your are getting.  \r\n\r\nIn case your are new to Lambda functions: anything you console.log will show up in AWS Cloudwatch assuming the Lambda function is executing under a role that allows it to write to Cloudwatch.  I think there is a built-in role like lambda-basic-execution or something.  You can set the role in the Lambda function under \"Execution role\" in the AWS Lambda console.  Once that is working, go to the AWS Cloudwatch console, click \"Logs\", select the /aws/lambda/{yourfunctionname} log, click \"Search Log Group\" and then I usually filter on the last five minutes of logs by clicking on \"5m\" in the upper right.\r\n\r\nIf you are still stuck let me know.\r\n\r\n-Mark"
    }
]